optimizer/index.js

```diff
@@ -33,7 +33,8 @@ const defaultConfig = {
     'ReorderHeadTransformer',
     // needs to run after ReorderHeadTransformer
     'RewriteAmpUrls',
-    'GoogleFontsPreconnect'
+    'GoogleFontsPreconnect',
+    'PruneDuplicatePreloads'
```

Comment:
suggestion: rename to `PruneDuplicateResourceHints`

Reply:
Done.